code/websites_and_search_engines.py

```diff
@@ -11,6 +11,11 @@
 )
 
 website_defaults = {
+    "talon website":"http://talonvoice.com",
+    "talon slack":"http://talonvoice.slack.com/messages/help",
+    "talon wiki":"https://talon.wiki/",
+    "talon practice":"https://chaosparrot.github.io/talon_practice/",
+    "talon repository search": "https://search.talonvoice.com/search/",
```

Comment:
maybe shorten a bit?
```suggestion
    "talon repo search": "https://search.talonvoice.com/search/",
```